container/docker/factory.go

```diff
@@ -236,6 +239,7 @@ func Register(factory info.MachineInfoFactory, fsInfo fs.FsInfo) error {
 		machineInfoFactory: factory,
 		storageDriver:      storageDriver(sd),
 		storageDir:         storageDir,
+		ignoreMetrics:      ignoreMetrics,
```

Comment:
(optional nit) I think this would all read a little more naturally if you inverted the logic (e.g. `includedMetrics` or `enabledMetrics`).